config/containers/logging/splunk.md

```diff
@@ -60,12 +62,12 @@ The following properties let you configure the splunk logging driver.
 | Option                      | Required | Description                                                                                                                                                                                                                                                                                                                                 |
 |:----------------------------|:---------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `splunk-token`              | required | Splunk HTTP Event Collector token.                                                                                                                                                                                                                                                                                                          |
-| `splunk-url`                | required | Path to your Splunk Enterprise, self-service Splunk Cloud instance, or Splunk Cloud managed cluster (including port and scheme used by HTTP Event Collector) in one of the following formats: `https://your_splunk_instance:8088` or `https://input-prd-p-XXXXXXX.cloud.splunk.com:8088` or `https://http-inputs-XXXXXXXX.splunkcloud.com`. |
+| `splunk-url`                | required | Path to your Splunk Enterprise, self-service Splunk Cloud instance, or Splunk Cloud managed cluster (including port and scheme used by HTTP Event Collector) in one of the following formats: `https://your_splunk_instance:8088`, `https://input-prd-p-XXXXXXX.cloud.splunk.com:8088`, or `https://http-inputs-XXXXXXXX.splunkcloud.com`.  |
```

Comment:
The table get cut in the browser view. Not sure if it's just my settings. Could you check?

![image](https://user-images.githubusercontent.com/47779042/78777490-882b9d80-7991-11ea-9338-c507dcc55bc1.png)

Reply:
It looks like the page isn't wide enough on 1280px wide

<img width="1292" alt="Screenshot 2020-04-08 at 13 55 50" src="https://user-images.githubusercontent.com/1804568/78781612-e1e79400-79a0-11ea-8a1e-b0ef66575773.png">



but it is working on either a wider window, or with the right-hand navigation collapsed:

<img width="1304" alt="Screenshot 2020-04-08 at 13 56 01" src="https://user-images.githubusercontent.com/1804568/78781618-e318c100-79a0-11ea-878c-25fa0b4a0f46.png">

I checked the same page on docs.docker.com, and it already has the same issue: https://docs.docker.com/config/containers/logging/splunk/

So it's not a regression at least